### ISSUE TITLE
Add alt attribute to pfp images on roster page

### ISSUE THREAD

**Issue by riley-spires** [ZipCode-UA/ZipCode-UA.github.io#11]
On the roster page (`public/pages/roster.html`), nobodies pfp has an alt tag which hurts both accessibility and makes the page a little worse if a pfp is missing (although we should never have this happen).
